Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/RawPixelReader.h

```diff
@@ -1108,7 +1108,7 @@ class RawPixelReader : public PixelReader
         rdhNew->triggerBC != rdhOld->triggerBC ||
         rdhNew->heartbeatOrbit != rdhOld->heartbeatOrbit ||
         rdhNew->heartbeatBC != rdhOld->heartbeatBC ||
-        rdhNew->triggerType != rdhOld->triggerType) {
+        !(rdhNew->triggerType & rdhOld->triggerType)) {
```

Comment:
@shahor02   Sorry, I am afraid I put this question at a wrong place. So I repeat it here:  What if the "new trigger" and "old trigger" differ  by just one bit. Naively, this should mean that these two triggers are  still different. This was the case with the previous logics. But the new  logics would consider such triggers as identical.  Is it indeed OK ?
--

Reply:
Hi @iouribelikov , the trigger is actually composed from different trigger bits, e.g. the HB trigger flag may coincide with the physical trigger flag PhT.  Right now I am not sure when the they will decouple: in the extra pages (RDH's) of the same physical trigger or at the next one, so I did it in the way that if they decouple after the 1st page, it is not interpreted as a new trigger. I assume once we understand all trigger states more complex check will be needed. Right not this check (especially in its old form) is not particularly useful since the read change of the trigger would necessarily change also the orbit/BC

Reply:
Thank you, @shahor02, for the clarification !  OK. Let's stay like that until the situation becomes more clear.